physics/sgscloud_radpre.F90

```diff
@@ -35,16 +35,19 @@ module sgscloud_radpre
 !!
 !>\section sgscloud_radpre_mod  SGS Cloud Scheme Pre General Algorithm
       subroutine sgscloud_radpre_run(    &
-           im,dt,levs,                   &
+           im,dt,fhswr,levs,             &
            flag_init,flag_restart,       &
            con_g, con_pi, eps, epsm1,    &
            r_v, cpv, rcp,                &
            xlv, xlf, cp,                 &
            do_mynnedmf,                  &
            qc, qi, qv, T3D, P3D, exner,  &
            qr, qs, qg,                   &
-           qci_conv,ud_mf,               &
+           qci_conv,qlc,qli,ud_mf,       &
+!           qci_conv_timeave,             &
```

Comment:
Any reason to leave these commented lines?

Reply:
I can remove them. They are really for an anticipated follow-up commit for the GF scheme. But they can be added at that point.